recipes-tag/intake-bluesky/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "0.1.0a8" %}
+{% set setup_py = load_setup_py_data() %}
+
+package:
+    name: 'intake-bluesky'
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/bluesky/intake-bluesky
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    script: pip install intake-bluesky
```

Is this correct? This seems like it could resolve to a remote repo.

---

I agree with @CJ-Wright that it should be changed to install the local package, like you did above with `pip install -ve .`

---

See my comment above.
```suggestion
    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```